scripts/build/shims/assert.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  ok() {},
+  strictEqual() {}
```

Comment:
We could make these actually work if we need to- where are they used?

Reply:
We should probably remove `assert` calls actually. It's used in places like `assert.ok(path instanceof FastPath)`.

Not sure these should go to production anyway

Reply:
I think those are currently removed in production, but I agree that we shouldn't rely on assert for those.

Reply:
They're not removed yet, that's why we need those shims

We should consider doing that in the future